fxprof-processed-profile/src/global_lib_table.rs

```diff
@@ -14,6 +15,11 @@ pub struct GlobalLibTable {
     used_libs: Vec<LibraryHandle>, // append-only for stable GlobalLibIndexes
     lib_map: FastHashMap<LibraryInfo, LibraryHandle>,
     used_lib_map: FastHashMap<LibraryHandle, GlobalLibIndex>,
+    /// We keep track of RVA addresses that exist in frames that are assigned to this
+    /// library, so that we can potentially provide symbolication info ahead of time.
+    /// This is here instead of in `LibraryInfo` because we don't want to serialize it,
+    /// and because it's currently a hack.
+    all_libs_seen_rvas: Vec<Option<BTreeSet<u32>>>,
```

Comment:
Please add a comment saying that the Vec is indexed by `LibraryHandle.0`.

Alternatively, you could make the Vec indexed by `GlobalLibIndex.0`, needing one less translation in `add_lib_used_rva` and at that point you could also remove the Option and always initialize it to the empty set.

Reply:
Yeah, good idea. FWIW, `GlobalLibIndex` is a confusing name, maybe just `UsedLibraryHandle`?

Reply:
`UsedLibraryHandle` sounds good